oca_port/migrate_addon.py

```diff
@@ -66,6 +66,26 @@
         f"\t\t=> {bc.BOLD}" "{new_pr_url}" f"{bc.END}",
     ]
 )
+RENAME_TIPS = "\n".join(
```

Comment:
I refactored the way tips are printed so it's more extendable and maintainable: https://github.com/OCA/oca-port/pull/65

This `RENAME_TIPS` can be replaced by a `MIG_RENAMED_TIPS` with required steps once it's merged.

Reply:
Above PR has been merged, you can rebase your work on top of it now